Makefile

```diff
@@ -531,6 +531,8 @@ cppcheck:
 	cppcheck -f --enable=warning --enable=portability \
 		--suppress="ctunullpointer" --suppress="nullPointer" \
 		--suppress="objectIndex" --suppress="comparePointers" \
+		--suppress="subtractPointers" --suppress="intToPointerCast" \
+		--check-level=exhaustive \
```

Comment:
OK fro the subtractPointers, but I'd keep cast warnings and fix casts instead

Reply:
I can't get a clean build with `make cppcheck` locally on Ubuntu 24 without this. But since CI isn't throwing this error, I will revert.

```
src/update_ram.c:179:28: portability: Casting non-zero decimal integer literal to pointer. [intToPointerCast]
            load_address = (uint32_t*)WOLFBOOT_LOAD_ADDRESS;
```